ibm_catalog.json

```diff
@@ -170,7 +170,27 @@
                   ]
                 },
                 {
-                  "key": "elasticsearch_version"
+                  "key": "elasticsearch_version",
+                  "required": true,
+                  "default_value": "",
+                  "options": [
+                    {
+                      "displayname": "v8.12 (preferred)",
```

Comment:
I would not add `preferred` here - as this may become out of date

Reply:
sure, done